bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -545,6 +552,10 @@
 		<label>Fan In (%)</label>
 		<description>Current relative speed (%) of incoming fan</description>
 		<category>Number</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Speed</tag>
```

Comment:
Not sure, if this is suitable. The values are more or less predefined for the set fan levels, so not really *measured*. And the percentage value doesn't really indicate `Speed`.
Would be better suited for `fan_in_RPM` and `fan_out_RPM`, IMO

Reply:
So probably `Status` then ??

Reply:
Nonetheless, `Measurement` + `Speed` could be added to `fan_in_RPM` and `fan_out_RPM`